tsdx.config.js

```diff
@@ -0,0 +1,32 @@
+// see: https://github.com/formium/tsdx#rollup
+
+// TODO: track https://github.com/formium/tsdx/issues/961 for possible better solution
+
+const path = require("path");
+
+const relativePath = p => path.join(__dirname, p);
+
+module.exports = {
+  rollup(config, options) {
+    if (options.format === "esm") {
```

Comment:
had to choose either `esm` or `cjs` and went with esm. Not really sure which should it be...